Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -246,11 +249,15 @@ static bool SendToObservers(NetworkIdentity identity, short msgType, MessageBase
                 // pack message into byte[] once
                 byte[] bytes = MessagePacker.Pack(msg);
```

Comment:
The goal is to get rid of this allocation

Reply:
the goal was to move to arraysegment afterwards. this PR just prepares for send(list)

Reply:
that means breaking the transport twice!

My PR #1122   does not break transports at all,  and it is based on your previous PR